src/components/utils/registerCustomKeysets.ts

```diff
@@ -0,0 +1,65 @@
+import type {KeysetData} from '@gravity-ui/i18n';
+
+import {i18n} from '../../i18n';
+
+export const registerCustomKeysets = (language: string, data: KeysetData) => {
+    validateCustomKeysets(language, data);
```

Comment:
I've decided to throw errors on startup, because this way developers will not have weird translation errors in their production, because we added new key to our component. I think that their CI or dev environment manual tests will easily catch it.

What do you guys, think?